<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFadingWindowClass</class>
 <widget class="QMainWindow" name="QFadingWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QFadingWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-60</y>
      <width>1141</width>
      <height>781</height>
     </rect>
    </property>
    <property name="viewMode">
     <enum>QMdiArea::TabbedView</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1126</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuFading">
    <property name="title">
     <string>Fading</string>
    </property>
    <addaction name="actionZbuffer"/>
    <addaction name="actionScanning_Line"/>
    <addaction name="actionSection_Zbuffer"/>
    <addaction name="actionNormal"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionChange_Color"/>
    <addaction name="actionSet_Color"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFading"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionNormal"/>
   <addaction name="actionZbuffer"/>
   <addaction name="actionScanning_Line"/>
   <addaction name="actionSection_Zbuffer"/>
   <addaction name="actionChange_Color"/>
   <addaction name="actionSet_Color"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/open.png</normaloff>../Icon/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionZbuffer">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/points.png</normaloff>../Icon/points.png</iconset>
   </property>
   <property name="text">
    <string>Zbuffer</string>
   </property>
  </action>
  <action name="actionScanning_Line">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/selected_edge.png</normaloff>../Icon/selected_edge.png</iconset>
   </property>
   <property name="text">
    <string>Scanning Line</string>
   </property>
  </action>
  <action name="actionSection_Zbuffer">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/cutingplane.png</normaloff>../Icon/cutingplane.png</iconset>
   </property>
   <property name="text">
    <string>Section Zbuffer</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/flat.png</normaloff>../Icon/flat.png</iconset>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionChange_Color">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/ball.png</normaloff>../Icon/ball.png</iconset>
   </property>
   <property name="text">
    <string>Single Color</string>
   </property>
  </action>
  <action name="actionChange">
   <property name="text">
    <string>change</string>
   </property>
  </action>
  <action name="actionSet_Color">
   <property name="icon">
    <iconset>
     <normaloff>../Icon/EntityItem.png</normaloff>../Icon/EntityItem.png</iconset>
   </property>
   <property name="text">
    <string>Random Color</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qfadingwindow.qrc"/>
 </resources>
 <connections/>
</ui>
